cfgmgr/vrfmgr.cpp

```diff
@@ -137,6 +140,41 @@ bool VrfMgr::setLink(const string& vrfName)
     return true;
 }
 
+void VrfMgr::handleVnetConfigSet(KeyOpFieldsValuesTuple &t)
+{
+    SWSS_LOG_ENTER();
+
+    string tunnelName;
+    vector<FieldValueTuple> tunnelValues;
+    vector<FieldValueTuple> vnetValues;
+
+    const auto& values = kfvFieldsValues(t);
+    const auto& vrfName = kfvKey(t);
+
+    for (const auto& fv : values)
+    {
+        const auto& name = fvField(fv);
+
+        if (name == "vni")
+        {
+            tunnelValues.push_back(fv);
+        }
+        else if (name == "tunnel")
+        {
+            tunnelName = fvField(fv);
+        }
+        else
+        {
+            vnetValues.push_back(fv);
+        }
+    }
+
+    tunnelValues.emplace_back("vrf", vrfName);
+
+    m_appVnetTableProducer.set(vrfName, vnetValues);
+    m_appTunnelMapTableProducer.set(tunnelName, tunnelValues);
```

Comment:
There is a tunnel map name as well to uniquely identify the the mapping. Can you check if its set?

```
VXLAN_TUNNEL_MAP:{{tunnel_name}}:{{**tunnel_map**}}
    "vni": {{ vni_id}}
    "vrf": {{ vrf_name }}
```
You can use the save Vnet/Vrf name for tunnel_map name as well.

Reply:
Done